notifications/notification.py

```diff
@@ -26,7 +26,8 @@ def run(self):
         try:
             server = smtplib.SMTP(settings.EMAIL_HOST, settings.EMAIL_PORT)
             server.ehlo()
-            server.starttls()
+            # NOTE: TLS is disabled for now, for alpha instance
+            # server.starttls()
```

Comment:
Shouldn't this also be configured through the environment variable?

Reply:
Let's add and use EMAIL_USE_TLS here (set True as default value to avoid changing production logic)